src/common/InputRowRadioButtons.js

```diff
@@ -0,0 +1,154 @@
+ */
+// This has the navigate-to-nested-screen semantics of NestedNavRow,
+// represented by IconRight. NestedNavRow would probably be the wrong
+// abstraction, though, because it isn't an imput component; it doesn't have
+// a value to display.
+export default function InputRowRadioButtons<TItemKey: string>(props: Props<TItemKey>): Node {
+  const { navigation, label, description, valueKey, items, onValueChange } = props;
+
+  const screenKey = useRef(
+    `selectable-options-${Math.floor(Math.random() * Number.MAX_SAFE_INTEGER).toString(36)}`,
+  ).current;
+
+  const selectedItem = items.find(c => c.key === valueKey);
+  invariant(selectedItem != null, 'InputRowRadioButtons: exactly one choice must be selected');
+
+  const screenParams = useMemo(
+    () => ({
+      title: label,
+      description,
+      items: items.map(({ key, title, subtitle }) => ({
+        key,
+        title,
+        subtitle,
+        selected: key === valueKey,
+      })),
+      onRequestSelectionChange: (itemKey, requestedValue) => {
+        if (!requestedValue || itemKey === valueKey) {
+          // Can't unselect a radio button.
+          return;
+        }
+        navigation.goBack();
+        onValueChange(itemKey);
+      },
+    }),
+    [navigation, label, description, items, valueKey, onValueChange],
+  );
+
+  const handleRowPressed = useCallback(() => {
+    // TODO: Can use .push instead of .navigate? See if Flow types are wrong
+    //   in saying we can't pass `key` when using .push.
+    navigation.navigate({
+      name: 'selectable-options',
+      key: screenKey,
+      params: screenParams,
+    });
+  }, [navigation, screenKey, screenParams]);
+
+  // Live-update the selectable-options screen.
+  useEffect(() => {
+    navigation.dispatch(
+      state =>
+        state.routes.find(route => route.key === screenKey)
+          ? { ...CommonActions.setParams(screenParams), source: screenKey }
+          : CommonActions.reset(state), // no-op
+    );
+  }, [navigation, screenParams, screenKey]);
+
+  // The desired height for IconRight, which we'll pass for its `size` prop.
+  // It'll also be its width.
+  const kRightArrowIconSize = 24;
+
+  const styles = useMemo(
+    () =>
+      createStyleSheet({
+        wrapper: {
+          flexDirection: 'row',
+          alignItems: 'center',
+          paddingVertical: 8,
+          paddingHorizontal: 16,
+        },
+        textWrapper: {
+          flex: 1,
+          flexDirection: 'column',
+        },
+        valueTitle: {
+          fontWeight: '300',
+          fontSize: 13,
+        },
+        iconRightWrapper: {
+          height: kRightArrowIconSize,
+
+          // IconRight is a square, so width equals height and this is the
+          // right amount of width to reserve.
+          width: kRightArrowIconSize,
+        },
+      }),
+    [],
+  );
+
+  return (
+    <Touchable onPress={handleRowPressed}>
+      <View style={styles.wrapper}>
```

Comment:
`Touchable` itself already creates a wrapper View. (This smooths over some variation between the underlying primitives RN gives us on iOS vs. Android; see the `Touchable` implementation.)

Can we use that one instead of having a second wrapper inside it? That'd look like:
```suggestion
    <Touchable onPress={handleRowPressed} style={styles.wrapper}>
```
(and drop a `</View>` at the end, and reindent.)

Reply:
Then the `<Touchable />` ends up with two children, which is apparently bad:

<img width=400 src="https://user-images.githubusercontent.com/22248748/167720623-f1d997bc-7fcd-4296-9ab7-affc348047ca.png" />